packages/neuron-ui/src/services/UILayer.ts

```diff
@@ -129,16 +134,34 @@ export const wallets = (
 }
 
 export const walletsCall = instantiateMethodCall(wallets) as {
-  index: () => void
+  getAll: () => void
+  get: (id: string) => void
   generateMnemonic: () => void
   importKeystore: (params: { name: string; keystore: string; password: string }) => void
   importMnemonic: (params: { name: string; mnemonic: string; password: string }) => void
   update: (params: { id: string; name?: string; password: string; newPassword?: string }) => void
   delete: (params: { id: string; password: string }) => void
   export: (id: string) => void
-  active: () => void
-  setActive: (id: string) => void
+  activeOne: () => void
```

Comment:
Does `activeOne` mean getting current active wallet? If so `getActive` would be easier to understand.

Reply:
updated in https://github.com/nervosnetwork/neuron/pull/223/commits/a2b0393e1cdf7689bde17b2497f31db4cd7e90fd